freetextresponse/freetextresponse.py

```diff
@@ -760,6 +759,35 @@ def save_reponse(self, data, suffix=''):
         }
         return result
 
+    @staticmethod
+    def custom_report_format(*args, **kwargs):
+        """
+        This method return the summited answer for the give user and block.
```

Comment:
I think that the name of the function and its description doesn't match, could you please change one of them?